README.md

```diff
@@ -25,12 +25,12 @@ import "@webcomponents/custom-elements/src/native-shim.js";
 import "@webcomponents/custom-elements/src/custom-elements.js";
 
 /* Import the host library */
-import host from "iframe-coordinator/host.js";
+import { registerElements } from "iframe-coordinator/host.js";
```

Comment:
This felt so much better to me in terms of a cleaner API.  It felt really odd to get an object just to call this one method.